### PR TITLE
Remove automatic capitalization of the strings (#434)

### LINKED ISSUE #434: text-transform:capitalize; do not works for all languages
The [CSS class](https://github.com/tmrowco/electricitymap/blob/master/web/public/css/styles.css#L215) `country-col-headline` contains following line:
```CSS
text-transform: capitalize;
```
It is there to make the first letter of each word to uppercase. This works well in English, but in Swedish it do not works at all (and I suspect it's the same problem in other languages to)

I suggest that this be removed. Please let the translators decide when it should be capitalized.

### PR DESCRIPTION
**Pull request by magol**
Let the translators to decide what is best for each language.

I have done changes in each translation file, so I want each translator to review it ans see if the capitalization is correct for each language.

| Language        | Translator |
| ------------- |-------------| 
|de | @timtroendle  |
|es | @bikiniweekend | 
|fr | @corradio, @kevindqc, @ThierryOllivero |
|it | @Raffox97 |
|nl | @beJels, @ThierryOllivero |
|sv | magol |

Fixes #434

### CODE REVIEW

File: web/locales/de.json [hunk 1]
```diff
@@ -51,11 +52,13 @@
     "legends": {
         "windpotential": "Potenzial Windenergie",
         "solarpotential": "Potenzial Solarenergie",
-        "colorblindmode": "farbenblind Modus"
+        "colorblindmode": "farbenblind Modus",
```
Comment (timtroendle) [inline web/locales/de.json]: I'd say `Modus für Farbenblinde`. It could be as well `Farbenblindmodus`, but I prefer the former.

File: web/locales/de.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
         "back": "zurück",
         "source": "Quelle",
         "fossilfuels": "fossile Brennstoffe",
+        "carbonintensity": "CO2 Konzentration",
```
Comment (timtroendle) [inline web/locales/de.json]: Actually, to correct myself: it should be rather `CO2-Konzentration`. Here and anywhere else.

Comment (magol) [inline web/locales/de.json]: Of course i can fix it, but I don't like to fix language issues of a language i don't understand. To easy to make it wrongly. 

I think this should be fixed in a separate PR so we not mix different type of changes.

Comment (corradio) [inline web/locales/de.json]: agreed. Also note that it should be intensity and not concentration.

File: web/locales/de.json [hunk 3]
```diff
@@ -66,7 +69,6 @@
         "withcarbonintensity": "mit einer CO2 Konzentration von"
     },
     "misc": {
-        "carbonintensity": "CO2 Konzentration",
```
Comment (corradio) [inline web/locales/de.json]: Should be kept

File: web/locales/de.json (hunk 2 above)
Comment (magol) [inline web/locales/de.json]: How do we do? Do you approve this PR and do you changes in a separate PR, or do you want to make any changes like **ThierryOllivero** did in 125647c?

Comment (corradio) [inline web/locales/de.json]: I'm all for merging fast so let's not wait for anything else, but on the other hand let's not spend time making multiple pull requests if other changes do not make this PR unreadable.

Comment (corradio) [inline web/locales/de.json]: Let me know if I should merge or wait for other changes.

Comment (magol) [inline web/locales/de.json]: Leta merge this as it is. No reson to wait 😊
